notify/notify.go

```diff
@@ -238,7 +238,7 @@ func createStage(rc *config.Receiver, tmpl *template.Template, wait func() time.
 		var s MultiStage
 		s = append(s, NewWaitStage(wait))
 		s = append(s, NewDedupStage(notificationLog, recv))
-		s = append(s, NewRetryStage(i))
+		s = append(s, NewRetryStage(i, recv))
```

Comment:
could you update this to only pass in the `GroupName`? No reason to pass around more data than is necessary.

Reply:
done